### ISSUE TITLE
Unify formatting by reinitiating Black

### ISSUE THREAD

**Issue by manzanillo** [eclipse-qrisp/Qrisp#85]
When working with the code I experienced a lot of different conventions being used when it comes to string indications ("" vs ''), spaces (e.g. def func1(param = "abc") vs  def func1(param="abc")). In order to avoid to create too much noise in PR for new contributors putting Black (and maybe isort) back into use could be very helpful.

**Comment by positr0nium** [comment]
@bastibock what do you think about this?

**Comment by bastibock** [comment]
I agree! This definitely makes sense. We tried integrating it before but had some issues, especially when integrating into the GitLab CI/CD pipeline. But since the repo is on GitHub now, we should integrate it here.
I will start working on it.

**Comment by positr0nium** [comment]
Fixed in 097274dd9740536c20da2bfd7d34965e57f82eec.
